internal/blockprocessor/processor.go

```diff
@@ -56,6 +56,16 @@ func New(conf *Config) *BlockProcessor {
 	}
 }
 
+// Bootstrap initializes the ledger and database with the first block, which contains a config transaction.
+// This block is a.k.a. the "genesis block".
+func (b *BlockProcessor) Bootstrap(configBlock *types.Block) error {
+	if err := b.initAndRecoverStateTrieIfNeeded(); err != nil {
+		panic(errors.WithMessage(err, "error while recovering node state trie"))
```

Comment:
`ValidateAndCommit()` panics because it is executing within a goroutine. For `Bootstrap()`, we can return the error.

Reply:
done